Makefile

```diff
@@ -317,6 +317,7 @@ presubmit_build: descriptions
 	$(GO) build ./...
 	$(MAKE) lint
 	SYZ_SKIP_DASHBOARD=1 $(MAKE) test
+	$(MAKE) presubmit_fuzzer
```

Comment:
How long does it take?
Is there a better presubmit section for this? This is not a "build".

Reply:
Locally it takes 5-7 seconds.

Reply:
`ci / build` still looks like the best choice for it.

If the coverage collection code begins to work well with gcc, it might not be needed to run it separately as it would work without `SYZ_CLANG=1`. However, we might still want to run this (and maybe a few other?) tests under clang, so something like `ci / build-clang` might be an option.